php/mixphp-swoole/bin/swoole.php

```diff
@@ -0,0 +1,13 @@
+<?php
+require __DIR__ . '/../vendor/autoload.php';
+
+$vega = App\Vega::new();
+$http = new Swoole\Http\Server('0.0.0.0', 3000);
+$http->on('Request', $vega->handler());
+$http->set([
+    'enable_coroutine' => false,
+    'worker_num' => swoole_cpu_num() * 2,
+    'log_file' => '/dev/null',
+    'log_level' => 5,
```

Comment:
As I understand, this redirect logs to `/dev/null`. This way is not interoperable (working and `unix` but not `win`). Is there not a way this work on all systems ?